projects/melon-core/src/partials/modules/fs/_deleteFile.ts

```diff
@@ -0,0 +1,5 @@
+function _deleteFile(path: string) {
+    _$internalBinding["DeleteFile"](path);
+}
+
+export { _deleteFile }
```

Comment:
Not much of a fan of these repeated exports on a separate line, although I get it's the project's style guide.

Since these files always export 1 function anyway, using `export function _deleteFile(path: string) {` would save 2 lines for each file. I think it's more readable.

Reply:
Will be enqueued in #164